### PR TITLE
[Fluent] Adding ManageVMInParallel sample and fixing the DAG so that all compa…

### PR DESCRIPTION
**Pull request by anuchandy**
…risions are case-insensitive

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@anuchandy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (anuchan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManagement/Azure/Azure.sln [hunk 1]
```diff
@@ -60,6 +60,8 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "ManageRedis", "..\..\Resour
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Azure.Management.Fluent.Batch", "..\Batch\Microsoft.Azure.Management.Fluent.Batch\Microsoft.Azure.Management.Fluent.Batch.xproj", "{08BACF2F-33C9-498D-81DC-377291356F39}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "ManageVirtualMachinesInParallel", "..\..\ResourceManagement.Samples\Compute\ManageVirtualMachinesInParallel\ManageVirtualMachinesInParallel.xproj", "{3A1AA1AC-2B74-4EC6-A777-4A79023B05E1}"
+EndProject
```
Comment (hovsepm) [inline src/ResourceManagement/Azure/Azure.sln]: please revert changes to this file

File: src/ResourceManagement.Samples/Compute/ManageVirtualMachinesInParallel2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,240 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (hovsepm) [inline src/ResourceManagement.Samples/Compute/ManageVirtualMachinesInParallel2/Program.cs]: Please change this project name to ManageVirtualMachinesInParallelWithNetwork

**Review by hovsepm — CHANGES_REQUESTED** [review]

File: src/ResourceManagement.Samples/Compute/ManageVirtualMachinesInParallel2/Program.cs (hunk 2 above)
Comment (anuchandy) [inline src/ResourceManagement.Samples/Compute/ManageVirtualMachinesInParallel2/Program.cs]: @hovsepm  thanks, please take a look at the updated sample.

**Review by hovsepm — APPROVED** [review]

**Pull request merged**